### PR TITLE
ocis_ldap: Update to a more update OpenLDAP image

### LINKED ISSUE #5636: Ocis against ldap. login problem
CC @rhafer 

Steps: 
- docker pull owncloud/ocis - need get ocis with disabling/enabling feature
- start ocis with ldap https://github.com/owncloud/ocis/blob/master/deployments/examples/ocis_ldap/docker-compose.yml
- try to login as admin with correct password

Actual: Logon failed

ldap_server_log:
```
63f73ebe conn=1004 fd=12 ACCEPT from IP=172.29.0.5:59784 (IP=0.0.0.0:389)

63f73ebe conn=1004 op=0 BIND dn="cn=admin,dc=owncloud,dc=com" method=128

63f73ebe conn=1004 op=0 BIND dn="cn=admin,dc=owncloud,dc=com" mech=SIMPLE ssf=0

63f73ebe conn=1004 op=0 RESULT tag=97 err=0 text=

63f73ebe conn=1004 op=1 SRCH base="ou=users,dc=owncloud,dc=com" scope=2 deref=0 filter="(&(&(objectClass=inetOrgPerson)(objectClass=ownCloud)(!(?ownCloudUserEnabled=FALSE)))(uid=admin))"

63f73ebe conn=1004 op=1 SRCH attr=dn givenName ownclouduuid binary uid mail displayName sn

63f73ebe conn=1004 op=1 SEARCH RESULT tag=101 err=0 nentries=0 text=

63f73ebe conn=1004 fd=12 closed (connection lost)
```

### PR DESCRIPTION
**Pull request by rhafer**
Switch away from the unmaintained osixia image to the more uptodate bitnami image.

Also update the owncloud schema to include the latest changes
    
Fixes #5636

cc @ScharfViktor

### CODE REVIEW

**Comment by mmattel** [comment]
Questions:

* as `osixia/openldap` is replaced, is there also a replacement  for the `osixia/phpldapadmin` ?
* `osixia` is also present in `oc10_ocis_parallel/docker-compose.yml` - no change there?
(I just did a small grep in the examples directory to find occurrences)
* do we need to adapt helm charts?

**Comment by rhafer** [comment]
>     * the latest change of [osixia/phpLDAPadmin](https://github.com/osixia/docker-phpLDAPadmin) is from 2019 (as far I can see), but the latest changes of the [phpLDAPadmin source](https://github.com/leenooks/phpLDAPadmin) are 2 weeks ago...

Yeah, that's another topic. The phpLDAPadmin image seems also unmaintained. And we should find a solution for that as well. I'll open an issue for that.

>     * `osixia` is also present in `oc10_ocis_parallel/docker-compose.yml` - no change there?
>       (I just did a small grep in the examples directory to find occurrences)

The state of the example is currently unclear to me. It seems to be in a somewhat broken state. At least I am not able to get it running. So I'd rather not touch it and break it even more.

>     * do we need to adapt helm charts?

Unless I am severely mistaken our helm charts don't currently deploy any OpenLDAP instance themselves.

File: deployments/examples/ocis_ldap/docker-compose.yml [hunk 1]
```diff
@@ -101,36 +100,41 @@ services:
...
 
   ldap-server:
-    image: osixia/openldap:latest
+    image: bitnami/openldap:2.6
     networks:
       ocis-net:
-    command: --copy-service --loglevel debug
+    entrypoint: ["/bin/sh", "/opt/bitnami/scripts/openldap/docker-entrypoint-override.sh", "/opt/bitnami/scripts/openldap/run.sh" ]
     environment:
+      BITNAMI_DEBUG: true
```
Comment (ScharfViktor) [inline deployments/examples/ocis_ldap/docker-compose.yml:112]: do we need debugging?  if yes can we change as `BITNAMI_DEBUG: "true"`

**Comment by ScharfViktor** [comment]
Sorry for late testing. I had vacation day

Probably for you it works correct. I see that image bitnami/openldap doesn't support arm and it doesn't work for my mac. 
may be a problem for some users.

<br class="Apple-interchange-newline"> 
image

**Pull request merged**